packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -464,20 +466,25 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
         if (props.multiselectable) {
             return null;
         }
+
         const isItemSelected: boolean = state.selectedItemIndex !== 0;
+
+        // we give the trigger button the role of textbox in order to get
+        // because Aria expects elements with role of "combobox", as is the case
+        // the select parent, to own a textbox from which assistive technologies get
+        // the current value.
```

Comment:
This is a confusing read, can you take a stab at re-wording it a little?

Reply:
More concise -
        // we give the trigger button the role of "textbox"
        // because Aria expects elements with role of "combobox" to contain a textbox